### DISCUSSION TITLE
DateTimePicker getSize issue

### DISCUSSION THREAD

**Discussion by awixor** [mantinedev/mantine#4325] [Q&A] [answered]
I am trying to use DateTimePicker from [@mantine/dates](https://www.npmjs.com/package/@mantine/dates).
Using it breaks the app and throws this error: 
`export 'getSize' (imported as 'getSize') was not found in '@mantine/core'`

<img width="1634" alt="image" src="https://github.com/mantinedev/mantine/assets/121669644/0c8b8502-f6e7-449b-b21e-08a07751d7ba">

- `"@mantine/core": "^5.10.5",`
- `"@mantine/dates": "^6.0.11",`
- `"@mantine/hooks": "^5.10.5",`
- `"dayjs": "^1.11.7",`

**Comment by rtivital** [comment, marked as answer]
All `@mantine/*` packages must have the same version. Either downgrade `@mantine/dates` or upgrade other packages to `6.x`
